layouts/shortcodes/ww_dynamic_content.html

```diff
@@ -5,9 +5,14 @@
        "to_update": ["last_modified_gu", "last_modified_enteric"],
        "type": "dataset"
      },
+     {
+       "url": "https://blobserver.dc.scilifelab.se/blob/SLU_wastewater_data_v1.0.csv/info.json",
+       "to_update": ["last_modified_uppsala", "last_modified_slu_flu", "last_modified_slu_rsv"],
+       "type": "dataset"
+     },
      {
        "url": "https://blobserver.dc.scilifelab.se/blob/SLU_wastewater_data.csv/info.json",
-       "to_update": ["last_modified_uppsala", "last_modified_slu_flu"],
+       "to_update": ["histroic_covid_slu", "historic_slu_flu"],
```

Comment:
Since there historic ones are hard coded now we don't need this, also above I see file `v1.0` is being used